test/system/090-events.bats

```diff
@@ -291,3 +291,22 @@ EOF
     _events_container_create_inspect_data journald
     _events_container_create_inspect_data file
 }
+
+@test "event with too big labels" {
+    # test that events entries are not truncated when reading from syslog
+
+    skip_if_remote "remote does not support --events-backend"
```

Comment:
You almost certainly also want `skip_if_journald_unavailable`

Reply:
fixed